libs/common-utils/src/fractionUtils.ts

```diff
@@ -193,6 +193,11 @@ function reduce(fraction: Fraction): Fraction {
   let numerator = fraction.numerator
   let denominator = fraction.denominator
   let rest: JSBI
+
+  if (JSBI.equal(denominator, ZERO)) {
+    return new Fraction(JSBI.BigInt(0), JSBI.BigInt(1))
+  }
```

Comment:
Shouldn't be needed because we shouldn't have any fractions with `0` as denominator, but some asshole was creating 0/0 fractions... wonder who.
Anyway, this will now fix any fraction like that if we ever see one.